### PR TITLE
Add get_active_uniform

### PR DESCRIPTION
**Pull request by saurvs**
Part of https://github.com/servo/servo/issues/10397

---
This change is [Reviewable](https://reviewable.io/reviews/servo/gleam/74)

### CODE REVIEW

File: src/gl.rs [hunk 1]
```diff
@@ -984,6 +984,30 @@ pub fn get_active_attrib(program: GLuint, index: GLuint) -> (i32, u32, String) {
...
+    let mut size: i32 = 0;
+    let mut type_: u32 = 0;
+
+    unsafe {
+        ffi::GetActiveUniform(program, index, buf_size, &mut length, &mut size,
+                              &mut type_, name.as_mut_ptr() as *mut GLchar);
+    }
+
+    name.truncate(
+        if length > 0 {
```
Comment (emilio) [inline src/gl.rs]: Please put this condition in the same line in order to be consistent with the rest of the code.

**Comment by emilio** [comment]
Looks good, `r=me` with that change :)

**Comment by saurvs** [comment]
Should I also bump up the version to `0.2.13`?

**Comment by emilio** [comment]
Oh yes, that way we can publish it right away :)

**Comment by saurvs** [comment]
Done.

**Comment by emilio** [comment]
@bors-servo: r+

Thanks and sorry for the delay reviewing this! :)

**Comment by saurvs** [comment]
It's okay!

**Comment by saurvs** [comment]
Once the version is published I can open the PR for `webrender_traits`.

**Comment by emilio** [comment]
Published! :)

**Pull request merged**